command/sub8_missions/nodes/move_command

```diff
@@ -95,6 +129,8 @@ if __name__ == '__main__':
         help="Make end goal have zero roll and pitch.")
     parser.add_argument('-b', '--blind', action='store_true',
         help="Do not check if waypoint is safe.")
+    parser.add_argument('-gt', '--gteleport', type=float,nargs='+',
+        help="Teleport sub in Gazebo to coordinates. Does nothing if not in Gazebo.")
```

Comment:
you can add ```metavar='X Y Z'``` here to make it more clear from the help prompt how to enter the coordinates.